libbeat/outputs/elasticsearch/testing.go

```diff
@@ -38,14 +40,15 @@ func GetEsPort() string {
 }
 
 // GetTestingElasticsearch creates a test client.
-func GetTestingElasticsearch() *Client {
+func GetTestingElasticsearch(t *testing.T) *Client {
 	var address = "http://" + GetEsHost() + ":" + GetEsPort()
 	username := os.Getenv("ES_USER")
 	pass := os.Getenv("ES_PASS")
 	client := newTestClientAuth(address, username, pass)
 
 	// Load version number
-	client.Connect(3 * time.Second)
+	err := client.Connect(60 * time.Second)
+	assert.NoError(t, err)
```

Comment:
It would better to use `if err != nil { t.Fatal(e) }` so that the test doesn't run. `assert.NoError` is like `t.Error` in that the test keeps executing.